.github/workflows/pr-checks.yaml

```diff
@@ -107,6 +107,21 @@ jobs:
 
           make SKIP_REGISTRY_LOGIN=true -o registry-login test-acceptance-with-bundle
 
+      - name: Collect Kube resources
+        continue-on-error: true
+        run: |
+          mkdir -p ${TEST_RESULTS}
+          ns=operators
+          for res in pods deployments subscriptions; do
```

Comment:
how about that we do not list just a few things here, instead we can collect really all resources with:

```
kubectl get all -n $ns -o yaml > ${TEST_RESULTS}/resources.${ns}.yaml
```

Reply:
Good ides, I'll update it. Thx @pedjak

Reply:
@pedjak `kubectl get all` does not return all resources in the namespace, only selected ones such as `Pod`, `Service`, `Deployment`, `ReplicaSet`, `Job`. No custom resources including `Subscription` or `CatalogSource`.

To get really all resources in namespace, we should to something like [this](https://stackoverflow.com/questions/47691479/listing-all-resources-in-a-namespace):

```bash
ns=operators
for res in $(kubectl api-resources --verbs=list --namespaced -o name); do
   kubectl get ${res} --ignore-not-found -n ${ns} -o yaml > ${TEST_RESULTS}/${res}.${ns}.yaml;
done                                    
```

Reply:
I'm just a bit concerned whether we would expose any sensitive data (stored in `Secret` resources) that way.

Reply:
do we really have any sensitive data in our tests/test clusters?

Reply:
Well, I meant in general.
For example the `install.sh` script creates a secret `sbo-operator-secrets` with a docker config which is internal for the GH workflow. And to be honest, I'm not sure what is inside :)

Reply:
Other then that, I don't see any possibly sensitive data in secrets from tests. Any other idea?

Reply:
I just want to make sure that we do not expose anything we shouln't.

Reply:
well, you can filter out secrets - what I am asking myself is what kind of insights we can get by collecting these resources anyway?

Reply:
Well, those are meant for the case of a installation/deploment failure - to investigate what went wrong and those resources contain additional information that can help to underatand the failure - especially SBO's Pod and Deployment, as well as Subscription and CatalogSource, that is why those were in the orinal PR.

Reply:
to investigate tests we would need to collect resources from the test namespace, anyway

Reply:
+1 on collecting resources from the test namespace